resources/lang/en/store-layout.php

```diff
@@ -17,6 +17,7 @@
     'button:delete' => 'Delete',
     'button:confirm' => 'Confirm',
     'button:create' => 'Create',
+    'button:download-import-csv' => 'Download csv',
```

Comment:
I know this is how it was written in Figma, but as it is an acronym i think we want to write it in upper case.

```suggestion
    'button:download-import-csv' => 'Download CSV',
```

Reply:
Oh, i just saw the suggestion, I have pushed the change already. Thanks!!